addons/website_sale/models/res_config_settings.py

```diff
@@ -50,14 +50,20 @@ class ResConfigSettings(models.TransientModel):
         readonly=False,
     )
     salesteam_id = fields.Many2one(related='website_id.salesteam_id', readonly=False)
-    website_sale_prevent_zero_price_sale = fields.Boolean(
-        string="Prevent Sale of Zero Priced Product",
-        related='website_id.prevent_zero_price_sale',
+    website_sale_prevent_sale = fields.Boolean(
+        related='website_id.prevent_sale',
         readonly=False,
     )
-    website_sale_contact_us_button_url = fields.Char(
-        string="Button Url",
-        related='website_id.contact_us_button_url',
+    website_sale_prevent_sale_for = fields.Selection(
+        related='website_id.prevent_sale_for',
+        readonly=False,
+    )
+    website_sale_prevent_sale_for_categories = fields.Many2many(
+        related='website_id.prevent_sale_for_categories',
+        readonly=False,
+    )
+    website_sale_contact_us_link_url = fields.Char(
+        related='website_id.contact_us_link_url',
```

Comment:
I guess we could remove `website_sale` prefix from the fields and if we want add something then may be we could add online like `prevent_online_sale` something like that. wdyt? or only `prevent_sale` is also fine IMO